src/main/java/net/pms/encoders/FFmpegAudio.java

```diff
@@ -201,8 +201,9 @@ public ProcessWrapper launchTranscode(
 		}
 
 		// Streaming support
-		cmdList.add("-movflags");
-		cmdList.add("+faststart");
+//		cmdList.add("-movflags");
+//		cmdList.add("frag_keyframe+empty_moov");
+		
```

Comment:
@Sami32 I don't see anyone else saying anything, so it might just be me being slow. It looks to me like you just put English words into another language's sentences some times, making it impossible for me to figure out the true meaning of the statement.